README.rst

```diff
@@ -24,10 +24,14 @@ Resources
 ---------
 For more information about Chapel, please refer to the following resources:
 
-=====================  ========================================
+.. NOTE
+   If you are viewing this file locally, we recommend referring to
+   doc/README.rst for local references to documentation and resources.
+
+=====================  ==================================================================================
```

Comment:
I think the separator above (and below) would look better when read from the command-line if it fit within 80-columns (even though not all the entries below do).  Or at least if it were no longer than the longest entry below (though I'd probably still make it only 80 columns if that didn't break the .rst rendering in any way).